target/linux/pistachio/modules.mk

```diff
@@ -1,3 +1,27 @@
+define KernelPackage/sound-pistachio-soc
+  TITLE:=SoC Audio support for the Pistachio SOC
+  KCONFIG:= \
+	CONFIG_SND_SOC_IMG=y \
+	CONFIG_SND_MIPS=y \
+        CONFIG_SND_SOC_IMG=y \
+	CONFIG_SND_SOC_IMG_PISTACHIO \
+	CONFIG_SND_SOC_IMG_PISTACHIO_EVENT_TIMER \
```

Comment:
you will also need 
CONFIG_SND_SOC_IMG_I2S_IN
CONFIG_SND_SOC_IMG_I2S_OUT
CONFIG_SND_SOC_IMG_PARALLEL_OUT
CONFIG_SND_SOC_IMG_SPDIF_IN
CONFIG_SND_SOC_IMG_SPDIF_OUT